service/src/main/java/com/ge/predix/acs/config/GraphBeanDefinitionRegistryPostProcessor.java

```diff
@@ -24,11 +24,9 @@ public void postProcessBeanFactory(final ConfigurableListableBeanFactory beanFac
     @Override
     public void postProcessBeanDefinitionRegistry(final BeanDefinitionRegistry registry) throws BeansException {
         BeanDefinition resourceRepositoryBeanDefinition = new RootBeanDefinition(GraphResourceRepository.class);
-        registry.registerBeanDefinition("resourceRepository", resourceRepositoryBeanDefinition);
         registry.registerBeanDefinition("resourceHierarchicalRepository", resourceRepositoryBeanDefinition);
 
         BeanDefinition subjectRepositoryBeanDefinition = new RootBeanDefinition(GraphSubjectRepository.class);
-        registry.registerBeanDefinition("subjectRepository", subjectRepositoryBeanDefinition);
         registry.registerBeanDefinition("subjectHierarchicalRepository", subjectRepositoryBeanDefinition);
     }
 }
```

Comment:
@sanjeevchopra, @bhuvan-rk,
SubjectHierarchicalRepository and resourceHierarchicalRepository are not used anywhere so it seems like we don't have to register its beans.  However, PrivilegeManagementNoRollbackTest and PrivilegeManagementServiceImplTest fail if these registrations are removed.  Need to figure out what the dependency is and remove/change.